src/Symfony/Component/Validator/Constraints/Existence.php

```diff
@@ -20,6 +21,7 @@ abstract class Existence extends Composite
 {
     public array|Constraint $constraints = [];
 
+    #[HasNamedArguments]
```

Comment:
we deprecated support for constraints having a constructor without the `#[HasNamedArguments]` attribute in 7.3, but we added the constructor here only in 7.4 when we deprecated passing option arrays to the base constraint